### PR TITLE
Tooltip: update demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/Tooltip/Overview/description.md [hunk 1]
```diff
@@ -1 +1,16 @@
-The Tooltip component displays a&nbsp;tooltip for a&nbsp;specified element on&nbsp;the page. This demo shows how you can customize the tooltip and animate its appearance on&nbsp;the page.
\ No newline at end of file
+The Tooltip component displays a tooltip near an element on the page. To bind the Tooltip to the element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Tooltip/Overview/description.md]: To bind the Tooltip to **any** element...

```suggestion
The Tooltip component displays a tooltip near an element on the page. To bind the Tooltip to an element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property. 
```

File: JSDemos/Demos/Tooltip/Overview/description.md [hunk 2]
```diff
@@ -1 +1,16 @@
-The Tooltip component displays a&nbsp;tooltip for a&nbsp;specified element on&nbsp;the page. This demo shows how you can customize the tooltip and animate its appearance on&nbsp;the page.
\ No newline at end of file
+The Tooltip component displays a tooltip near an element on the page. To bind the Tooltip to the element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property. 
+
+## Show and Hide the Tooltip
+
+Use two-way binding on the [visible](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#visible) property of the Tooltip and assign a variable to this property. In the target element markup, assign a function that changes this variable to the specific [DOM events](https://developer.mozilla.org/en-US/docs/Web/Events#event_listing) that trigger the visibility of a tooltip.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Tooltip/Overview/description.md]: > In the target element markup, assign a function that changes this variable to the specific [DOM events](https://developer.mozilla.org/en-US/docs/Web/Events#event_listing) that trigger the visibility of a tooltip.

I cannot say I understand this sentence:
1. A function that changes this variable to the specific DOM events. Does the function replace the variable with specific DOM events?
2. DOM events that trigger the visibility of a tooltip. What do you mean by "trigger the visibility"?

File: JSDemos/Demos/Tooltip/Overview/description.md [hunk 3]
```diff
@@ -1 +1,16 @@
...
+
+To hide the Tooltip when a user clicks outside it, use the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#hideOnOutsideClick) property.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Tooltip/Overview/description.md]:
```suggestion
To hide the Tooltip when a user clicks outside its borders, use the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#hideOnOutsideClick) property.
```

File: JSDemos/Demos/Tooltip/Overview/description.md [hunk 4]
```diff
@@ -1 +1,16 @@
...
+
+## Customize and Animate the Tooltip
+
+Assign Tooltip's content in the HTML markup. You can also use [contentTemplate](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#contentTemplate) to customize the Tooltip's content.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Tooltip/Overview/description.md]: Please do not use API members as ordinary sentence members. And don't forget about articles.

```suggestion
Assign the Tooltip's content in the HTML markup. Alternatively, you can use the [content template](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#contentTemplate) to customize the Tooltip's content.
```

File: JSDemos/Demos/Tooltip/Overview/description.md [hunk 5]
```diff
@@ -1 +1,16 @@
...
+
+If you need to position the Tooltip against a certain side of the target element, specify the [position](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#position) property.
+
+To animate the Tooltip, declare the [animation](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/) object. In this object, specify [show](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/#show) and [hide](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/#hide) fields.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Tooltip/Overview/description.md]: Articles.

```suggestion
To animate the Tooltip, declare the [animation](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/) object. In this object, specify the [show](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/#show) and [hide](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/#hide) fields.
```

File: JSDemos/Demos/Tooltip/Overview/jQuery/description.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+The Tooltip component displays a tooltip near an element on the page. To bind the Tooltip to the element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Tooltip/Overview/jQuery/description.md]: To bind the Tooltip to **any** element.

```suggestion
The Tooltip component displays a tooltip near an element on the page. To bind the Tooltip to an element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property. 
```

File: JSDemos/Demos/Tooltip/Overview/jQuery/description.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+The Tooltip component displays a tooltip near an element on the page. To bind the Tooltip to the element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property. 
+
+## Show and Hide the Tooltip
+
+To show and hide the Tooltip when certain events are triggered, set the [showEvent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/showEvent/) and [hideEvent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/hideEvent/) properties. These properties can accept several events at once as well as an object.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Tooltip/Overview/jQuery/description.md]: "Several" is a colloquialism. Use "multiple" instead. These words have identical meaning, however, "multiple" is more suitable to documentation.

```suggestion
To show and hide the Tooltip in response to certain events, specify the [showEvent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/showEvent/) and [hideEvent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/hideEvent/) properties. These properties can accept multiple events at once as well as an object.
```

File: JSDemos/Demos/Tooltip/Overview/jQuery/description.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+
+To hide the Tooltip when a user clicks outside it, use the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#hideOnOutsideClick) property.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Tooltip/Overview/jQuery/description.md]:
```suggestion
To hide the Tooltip when a user clicks outside its borders, use the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#hideOnOutsideClick) property.
```

File: JSDemos/Demos/Tooltip/Overview/jQuery/description.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+
+## Customize and Animate the Tooltip
+
+Assign Tooltip's content in the HTML markup. You can also use [contentTemplate](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#contentTemplate) to customize the Tooltip's content.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Tooltip/Overview/jQuery/description.md]: Same as above.

```suggestion
Assign the Tooltip's content in the HTML markup. Alternatively, you can use the [content template](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#contentTemplate) to customize the Tooltip's content.
```

File: JSDemos/Demos/Tooltip/Overview/jQuery/description.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+
+If you need to position the Tooltip against a certain side of the target element, specify the [position](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#position) property.
+
+To animate the Tooltip, declare the [animation](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/) object. In this object, specify [show](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/#show) and [hide](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/#hide) fields.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Tooltip/Overview/jQuery/description.md]: Articles.

```suggestion
To animate the Tooltip, declare the [animation](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/) object. In this object, specify the [show](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/#show) and [hide](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/animation/#hide) fields.
```

File: JSDemos/Demos/Tooltip/Overview/description.md (hunk 2 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/Tooltip/Overview/description.md]: How about this (I rewrote the whole paragraph)?

> Specify [DOM events](https://developer.mozilla.org/en-US/docs/Web/Events#event_listing) in the target element markup that need to be triggered to change the Tooltip visibility. Bind the [visible](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#visible) property of the Tooltip to a variable and use event handlers to toggle this variable.

Comment (dmitry-eliseev-devexpress) [reply JSDemos/Demos/Tooltip/Overview/description.md]: Now this sentence makes sense. Thank you.

File: JSDemos/Demos/Tooltip/Overview/description.md [hunk 11]
```diff
@@ -1 +1,16 @@
-The Tooltip component displays a&nbsp;tooltip for a&nbsp;specified element on&nbsp;the page. This demo shows how you can customize the tooltip and animate its appearance on&nbsp;the page.
\ No newline at end of file
+The Tooltip component displays a tooltip near an element on the page. To bind the Tooltip to an element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property. 
+
+## Show and Hide the Tooltip
+
+Specify [DOM events](https://developer.mozilla.org/en-US/docs/Web/Events#event_listing) in the target element markup that need to be triggered to change the Tooltip visibility. Bind the [visible](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#visible) property of the Tooltip to a variable and use event handlers to toggle this variable.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Tooltip/Overview/description.md]:
```suggestion
Specify [DOM events](https://developer.mozilla.org/en-US/docs/Web/Events#event_listing) in the target element's markup that needs to be triggered to change the Tooltip's visibility. Bind the [visible](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#visible) property of the Tooltip to a variable and use event handlers to toggle this variable.
```

File: JSDemos/Demos/Tooltip/Overview/description.md [hunk 12]
```diff
@@ -1 +1,16 @@
-The Tooltip component displays a&nbsp;tooltip for a&nbsp;specified element on&nbsp;the page. This demo shows how you can customize the tooltip and animate its appearance on&nbsp;the page.
\ No newline at end of file
+The Tooltip component displays a tooltip near an element on the page. To bind the Tooltip to an element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Tooltip/Overview/description.md]: Do you mean 
```suggestion
The Tooltip component can display a tooltip for an element on the page. To bind the Tooltip to an element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property. 
```

File: JSDemos/Demos/Tooltip/Overview/description.md [hunk 13]
```diff
@@ -1 +1,16 @@
...
+
+Specify [DOM events](https://developer.mozilla.org/en-US/docs/Web/Events#event_listing) in the target element markup that need to be triggered to change the Tooltip visibility. Bind the [visible](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#visible) property of the Tooltip to a variable and use event handlers to toggle this variable.
+
+To hide the Tooltip when a user clicks outside its borders, use the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#hideOnOutsideClick) property.
+
+## Customize and Animate the Tooltip
+
+Assign the Tooltip's content in the HTML markup. Alternatively, you can use the [content template](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#contentTemplate) to customize the Tooltip's content.
+
+If you need to position the Tooltip against a certain side of the target element, specify the [position](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#position) property.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Tooltip/Overview/description.md]: What do you mean by `against a certain side`? 
```suggestion
If you need to position the Tooltip at a certain side of the target element, specify the [position](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#position) property.
```

**Review by dirk-pieterse — DISMISSED** [review]

File: JSDemos/Demos/Tooltip/Overview/jQuery/description.md [hunk 14]
```diff
@@ -0,0 +1,15 @@
+The Tooltip component displays a tooltip near an element on the page. To bind the Tooltip to an element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property.
```
Comment (vladaskorohodova) [inline JSDemos/Demos/Tooltip/Overview/jQuery/description.md]:
```suggestion
The Tooltip component can display a tooltip for an element on the page. To bind the Tooltip to an element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property. 
```

File: JSDemos/Demos/Tooltip/Overview/jQuery/description.md [hunk 15]
```diff
@@ -0,0 +1,15 @@
...
+
+To show and hide the Tooltip in response to certain events, specify the [showEvent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/showEvent/) and [hideEvent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/hideEvent/) properties. These properties can accept multiple events at once as well as an object.
+
+To hide the Tooltip when a user clicks outside its borders, use the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#hideOnOutsideClick) property.
+
+## Customize and Animate the Tooltip
+
+Assign the Tooltip's content in the HTML markup. Alternatively, you can use the [content template](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#contentTemplate) to customize the Tooltip's content.
+
+If you need to position the Tooltip against a certain side of the target element, specify the [position](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#position) property.
```
Comment (vladaskorohodova) [inline JSDemos/Demos/Tooltip/Overview/jQuery/description.md]:
```suggestion
If you need to position the Tooltip at a certain side of the target element, specify the [position](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#position) property.
```

File: JSDemos/Demos/Tooltip/Overview/description.md [hunk 16]
```diff
@@ -1 +1,16 @@
-The Tooltip component displays a&nbsp;tooltip for a&nbsp;specified element on&nbsp;the page. This demo shows how you can customize the tooltip and animate its appearance on&nbsp;the page.
\ No newline at end of file
+The Tooltip component can display a tooltip for an element on the page. To bind the Tooltip to an element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property. 
+
+## Show and Hide the Tooltip
+
+Specify [DOM events](https://developer.mozilla.org/en-US/docs/Web/Events#event_listing) in the target element's markup that needs to be triggered to change the Tooltip's visibility. Bind the [visible](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#visible) property of the Tooltip to a variable and use event handlers to toggle this variable.
```
Comment (vladaskorohodova) [inline JSDemos/Demos/Tooltip/Overview/description.md]:
```suggestion
Specify [DOM events](https://developer.mozilla.org/en-US/docs/Web/Events#event_listing) in the target element's markup that need to be triggered to change the Tooltip's visibility. Bind the [visible](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#visible) property of the Tooltip to a variable and use event handlers to toggle this variable.
```

**Pull request merged**